load/creator.go

```diff
@@ -5,10 +5,13 @@ package load
 type DBCreator interface {
 	// Init should set up any connection or other setup for talking to the DB, but should NOT create any databases
 	Init()
+
```

Comment:
Not sure this is necessary?

Reply:
More clearly separates methods in the interface, does it not? At least when there are many methods it helps to read easier (for me at least).